spec/models/user_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { build(:user) }
+
+  describe 'validations' do
+    before { create(:user) }
```

Comment:
I must have missed something. But where did we actually use the user that we created in this line for this `describe "validations" block`?